vivisect/tools/graphutil.py

```diff
@@ -15,26 +15,39 @@
 
 xrskip = envi.BR_PROC | envi.BR_DEREF
 
-def getLongPath(g, maxpath=1000):
-    '''
-    Returns a list of list tuples (node id, edge id) representing the longest path
+def getNodeWeightHisto(g):
```

Comment:
this functionality is used in multiple places, and is valuable to have split out.